CONTRIBUTING.md

```diff
@@ -40,8 +48,8 @@ cargo fmt --all
 For the static analysis, we use [`clippy`].
 
 ```sh
-# To format the source code in the entire repository
-cargo clippy --workspace
+# Check lints on the source code in the entire repository
+cargo clippy --workspace --all-features --all-targets -- --warn clippy::cargo --allow clippy::multiple_crate_versions --deny warnings
```

Comment:
why not put this on Makefile?

Reply:
Yes, it's spotted. To be addressed in a future PR!